README.md

```
@@ -93,12 +101,21 @@ errlens run app.js --lang es
+errlens run <file> [options]              # Run a script and analyze any crashes
+errlens analyze <error> [options]         # Analyze a specific error message
+errlens --version                         # Show version information
+errlens --help                            # Show help
 ```
 
+### Global Options
+
+| Flag | Description |
+|------|-------------|
+| `--json` | Output raw JSON (no colors, no boxes) — perfect for CI/CD pipelines |
+| `--theme <name>` | Pick a color theme (see [Color Themes](#-color-themes) below) |
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)